src/server/core/webserver.ts

```diff
@@ -96,7 +96,11 @@ export class WebServer extends EventEmitter implements BaseKrasServer {
     this.server = ssl ? createHttpsServer(ssl, this.app) : createHttpServer(this.app);
     this.wsOptions = typeof config.ws === 'object' ? config.ws : undefined;
     this.ws = !!config.ws;
-    const upload = multer({ storage: multer.memoryStorage(), limits: { files: 5, fileSize: 10 * 1024 * 1024 } });
+    const fileSizeLimit = (process.env.FILE_SIZE_LIMIT && parseInt(process.env.FILE_SIZE_LIMIT, 10)) || 10;
```

Comment:
Looks good. We could also put it in `config`, where the default value would either be `process.env.FILE_SIZE_LIMIT` or `10 * 1024 * 1024`. I'll make that enhancement and document it!